README.md

```diff
@@ -1,2 +1,90 @@
 # services
 Overview of all OpenML components including a docker-compose to run OpenML services locally
+
+## Overview
+
+![OpenML Component overview](./documentation/OpenML-overview.png)
```

Comment:
The programmatic interfaces also communicate with the authorization in the server.

Reply:
Distinction between "server" and "backend" is going to be vague. In this sense it might even be more meaningful to change `minio` with `server cron jobs`, as then the two server layers are distinct in _software we wrote_ and _software we did not write_.

Reply:
Both point fixed in https://github.com/openml/services/pull/3/commits/070323b6d66cbe2ab5a490a71f1d0d6a9d7a452c. 
I removed the distinction between "server" and "backend", and added an "auth" arrow between the LIBRARIES and Backend